ton_sdk/src/lib.rs

```diff
@@ -72,45 +52,31 @@ pub use local_tvm::executor::TransactionFees;
 mod transaction;
 #[cfg(feature = "node_interaction")]
 pub use transaction::*;
-/*
-#[cfg(feature = "node_interaction")]
-mod block;
-#[cfg(feature = "node_interaction")]
-pub use block::*;
-*/
+
 mod types;
 pub use types::*;
 
 #[cfg(feature = "node_interaction")]
-pub mod queries_helper;
+pub mod node_client;
+#[cfg(feature = "node_interaction")]
+pub use node_client::*;
```

Comment:
Please, list types you import. ::* is now prohibited in tonlabs

Reply:
Done